doc/man3/RAND_bytes.pod

```diff
@@ -19,8 +19,10 @@ Deprecated:
 
 =head1 DESCRIPTION
 
+These functions are part of the OpenSSL L<RAND(7)> API.
+
```

Comment:
Wait, what?  That was rather unusual...  and considering the names of the functions described, this comes as no surprise.  So this becomes a bit over the top.  I'd say it's sufficient to add `L<RAND(7)>` in the `SEE ALSO` section instead.

Reply:
that goes for *all* places where you've added this sentense...